code/obj/flock/cage.dm

```diff
@@ -262,8 +262,9 @@
 		if(ON_COOLDOWN(src,"move_damage",1 SECOND))
 			return
 		if(prob(75))
```

Comment:
I'd probably add a separate cooldown here for the messages, otherwise it might get a little spammy with a message every second.